docs/ECLLanguageReference/ECLR_mods/Value-QString.xml

```diff
@@ -23,6 +23,8 @@
   to the size needed to contain the result of a cast or passed parameter. You
   may use set indexing into any QSTRING to parse out a substring.</para>
 
+  <para>The upper size limit for any QSTRING value is 4GB.</para>
```

Comment:
Technically the maximum length if 4G (=> maximum size is 3Gb), but I will merge anyway